modules/stochastic_tools/src/reporters/AdaptiveMonteCarloDecision.C

```diff
@@ -76,6 +79,16 @@ AdaptiveMonteCarloDecision::AdaptiveMonteCarloDecision(const InputParameters & p
   }
```

Comment:
I cannot comment on the part before, but 
```
_communicator.split(
_sampler.getNumberOfLocalRows() > 0 ? 1 : MPI_UNDEFINED, processor_id(), _local_comm);
```
this communicator is already in the sampler. You should only add an accessor function.

Reply:
Can you please tell me how to do this?

Reply:
You know how to do this now :). The necessary code should be in next from your other PR.

Reply:
The necessary code should be in next from your other PR!

Reply:
Yes, I will address this after that PR is merged.